security/advancedtls/examples/examples_test.sh

```diff
@@ -55,7 +55,8 @@ EXAMPLES=(
 
 declare -a EXPECTED_SERVER_OUTPUT=("Client common name: foo.bar.hoo.com" "Client common name: foo.bar.another.client.com")
 
-declare -a EXPECTED_CLIENT_OUTPUT=("Client stops sending requests")
+declare -a EXPECTED_SERVER_EXIT_OUTPUT=("signal: terminated")
+declare -a EXPECTED_CLIENT_EXIT_OUTPUT=("signal: terminated")
```

Comment:
This may not be very portable.  Is it needed?

Reply:
Sure, I will delete this line. I wanted to make sure the client and server are both terminated at the end, but it's not necessary.